app/components/Root.jsx

```diff
@@ -1,115 +1,56 @@
 import React, { Component } from 'react';
-
-export default class WinterJokes extends Component {
+import { connect } from 'react-redux';
+import Navbar from './Navbar';
+import Home from './Home';
+import {Route, Switch} from 'react-router-dom';
+import {Router} from 'react-router'
+import history from './history'
+import { getStudentsThunkCreator, getCampusesThunkCreator } from '../reducers/index.jsx'
+import store from '../store.jsx'
+import StudentListContainer from './Students'
+import CampusListContainer from './Campuses'
+import IndividualCampusContainer from './IndividualCampus'
+import IndividualStudent from './IndividualStudent'
+// import { render } from 'react-dom'
+
+
+class Root extends Component {
   constructor() {
     super()
-    this.nextJoke = this.nextJoke.bind(this)
-    this.answer = this.answer.bind(this)
-  }
-
-  componentDidMount() {
-    this.nextJoke()
   }
 
-  nextJoke() {
-    this.setState({
-      joke: randomJoke(),
-      answered: false,
-    })
-  }
+  componentDidMount(){
+    const getStudentsThunk = getStudentsThunkCreator();
+    store.dispatch(getStudentsThunk)
+    const getCampusesThunk = getCampusesThunkCreator();
+    store.dispatch(getCampusesThunk)
```

Comment:
Looks like these never got wired up to the dispatches provided by `mapDispatchToProps`.